### PR TITLE
Black formatting is missing for documentation

### PR DESCRIPTION
**Pull request by awav**
Subj.

### CODE REVIEW

**Comment by st--** [comment]
Have you checked that all notebooks are still correctly building after these changes? While I trust that the reformatting doesn't affect the python execution, I'm slightly less sure about how it works with jupytext.

**Comment by awav** [comment]
> Have you checked that all notebooks are still correctly building after these changes? While I trust that the reformatting doesn't affect the python execution, I'm slightly less sure about how it works with jupytext.

Yep, that looks fine.

**Comment by awav** [comment]
> Could you please rebase your branch and cherry-pick the latest three commits? It looks like you were on a stale branch when you created this branch.

Develop is up-to-date on my laptop.

```bash
(tf) artemav:~/code/GPflow/doc/source/notebooks (develop)
→ git pull
Already up to date.
```

**Comment by st--** [comment]
Not convinced by the double blank lines in notebook code cells, but I guess I can live with that...

**Comment by st--** [comment]
It's also missing for `setup.py` if you want to be comprehensive ?

File: doc/source/notebooks/advanced/kernels.pct.py [hunk 1]
```diff
@@ -200,16 +208,16 @@ def plotkernelfunction(k, ax, xmin=-3, xmax=3, other=0):
...
-Z = np.random.multivariate_normal(np.zeros(n_grid**2), K, 2)
-axes[1].imshow(Z[0, :].reshape(n_grid, n_grid), extent=[-10,10,-10,10])
-axes[1].set_title('GP sample 1')
-axes[2].imshow(Z[1, :].reshape(n_grid, n_grid), extent=[-10,10,-10,10])
-axes[2].set_title('GP sample 2');
+Z = np.random.multivariate_normal(np.zeros(n_grid ** 2), K, 2)
+axes[1].imshow(Z[0, :].reshape(n_grid, n_grid), extent=[-10, 10, -10, 10])
+axes[1].set_title("GP sample 1")
+axes[2].imshow(Z[1, :].reshape(n_grid, n_grid), extent=[-10, 10, -10, 10])
+axes[2].set_title("GP sample 2")
```
Comment (awav) [inline doc/source/notebooks/advanced/kernels.pct.py]:
```suggestion
_ = axes[2].set_title("GP sample 2")
```

File: doc/source/notebooks/advanced/varying_noise.pct.py [hunk 2]
```diff
@@ -138,13 +152,20 @@ def objective_closure():
...
+plt.errorbar(
+    X.squeeze(),
+    Y.squeeze(),
+    yerr=2 * (np.sqrt(NoiseVar)).squeeze(),
+    marker="x",
+    lw=0,
+    elinewidth=1.0,
+    color="C1",
+)
+plt.xlim(-5, 5)
```
Comment (awav) [inline doc/source/notebooks/advanced/varying_noise.pct.py]:
```suggestion
_ = plt.xlim(-5, 5)
```

File: doc/source/notebooks/advanced/varying_noise.pct.py [hunk 3]
```diff
@@ -225,12 +249,12 @@ def objective_closure():
...
-ax.plot(xx, mu + 2*np.sqrt(var), 'C0', lw=0.5)
-ax.plot(xx, mu - 2*np.sqrt(var), 'C0', lw=0.5)
+ax.plot(xx, mu, "C0")
+ax.plot(xx, mu + 2 * np.sqrt(var), "C0", lw=0.5)
+ax.plot(xx, mu - 2 * np.sqrt(var), "C0", lw=0.5)
 
-ax.plot(X, Y, 'C1x', mew=2)
-ax.set_xlim(-5, 5);
+ax.plot(X, Y, "C1x", mew=2)
+ax.set_xlim(-5, 5)
```
Comment (awav) [inline doc/source/notebooks/advanced/varying_noise.pct.py]:
```suggestion
_ = ax.set_xlim(-5, 5)
```

File: doc/source/notebooks/advanced/varying_noise.pct.py [hunk 4]
```diff
@@ -270,14 +297,15 @@ def objective_closure():
...
-ax.plot(xx, mu - 2*np.sqrt(var), 'C0', lw=0.5)
+ax.plot(xx, mu, "C0")
+ax.plot(xx, mu + 2 * np.sqrt(var), "C0", lw=0.5)
+ax.plot(xx, mu - 2 * np.sqrt(var), "C0", lw=0.5)
 
-ax.plot(X, Y, 'C1x', mew=2)
+ax.plot(X, Y, "C1x", mew=2)
 ax.set_xlim(-5, 5)
-ax.plot(xx, 2.5 * np.sin(6 * xx) + np.cos(3 * xx), 'C2--');
+ax.plot(xx, 2.5 * np.sin(6 * xx) + np.cos(3 * xx), "C2--")
```
Comment (awav) [inline doc/source/notebooks/advanced/varying_noise.pct.py]:
```suggestion
_ = ax.plot(xx, 2.5 * np.sin(6 * xx) + np.cos(3 * xx), "C2--")
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 5]
```diff
@@ -63,8 +64,8 @@
 
 L = np.linalg.cholesky(K)
 f_grid = np.dot(L, rng.randn(200, 5))
-plt.plot(x_grid, f_grid, 'C0', linewidth=1)
-plt.plot(x_grid, f_grid[:, 1], 'C0', linewidth=2);
+plt.plot(x_grid, f_grid, "C0", linewidth=1)
+plt.plot(x_grid, f_grid[:, 1], "C0", linewidth=2)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(x_grid, f_grid[:, 1], "C0", linewidth=2)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 6]
```diff
@@ -75,9 +76,10 @@
 def invlink(f):
     return gpflow.likelihoods.Bernoulli().invlink(f).numpy()
 
+
 p_grid = invlink(f_grid)
-plt.plot(x_grid, p_grid, 'C1', linewidth=1)
-plt.plot(x_grid, p_grid[:, 1], 'C1', linewidth=2);
+plt.plot(x_grid, p_grid, "C1", linewidth=1)
+plt.plot(x_grid, p_grid[:, 1], "C1", linewidth=2)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(x_grid, p_grid[:, 1], "C1", linewidth=2)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 7]
```diff
@@ -146,54 +151,61 @@ def objective():
...
-mask = Y[:, 0]==1
+X = np.loadtxt("data/banana_X_train", delimiter=",")
+Y = np.loadtxt("data/banana_Y_train", delimiter=",").reshape(-1, 1)
+mask = Y[:, 0] == 1
 
 plt.figure(figsize=(6, 6))
-plt.plot(X[mask, 0], X[mask, 1], 'oC0', mew=0, alpha=0.5)
-plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], 'oC1', mew=0, alpha=0.5);
+plt.plot(X[mask, 0], X[mask, 1], "oC0", mew=0, alpha=0.5)
+plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], "oC1", mew=0, alpha=0.5)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], "oC1", mew=0, alpha=0.5)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 8]
```diff
@@ -208,11 +220,18 @@ def objective():
 
 p, _ = m.predict_y(Xplot)  # here we only care about the mean
 plt.figure(figsize=(7, 7))
-plt.plot(X[mask, 0], X[mask, 1], 'oC0', mew=0, alpha=0.5)
-plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], 'oC1', mew=0, alpha=0.5);
-
-plt.contour(xx, yy, p.numpy().reshape(*xx.shape), [0.5],  # plot the p=0.5 contour line only
-            colors='k', linewidths=1.8, zorder=100);
+plt.plot(X[mask, 0], X[mask, 1], "oC0", mew=0, alpha=0.5)
+plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], "oC1", mew=0, alpha=0.5)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(X[np.logical_not(mask), 0], X[np.logical_not(mask), 1], "oC1", mew=0, alpha=0.5)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 9]
```diff
@@ -208,11 +220,18 @@ def objective():
...
+
+plt.contour(
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.contour(
```

File: doc/source/notebooks/basics/regression.pct.py [hunk 10]
```diff
@@ -38,18 +38,18 @@
...
 # `X` and `Y` denote the input and output values. **NOTE:** `X` and `Y` must be two-dimensional NumPy arrays, $N \times 1$ or $N \times D$, where $D$ is the number of input dimensions/features, with the same number of rows as $N$ (one for each data point):
 
 # %%
-data = np.genfromtxt('data/regression_1D.csv', delimiter=',')
+data = np.genfromtxt("data/regression_1D.csv", delimiter=",")
 X = data[:, 0].reshape(-1, 1)
 Y = data[:, 1].reshape(-1, 1)
 
-plt.plot(X, Y, 'kx', mew=2);
+plt.plot(X, Y, "kx", mew=2)
```
Comment (awav) [inline doc/source/notebooks/basics/regression.pct.py]:
```suggestion
_ = plt.plot(X, Y, "kx", mew=2)
```

File: doc/source/notebooks/basics/regression.pct.py [hunk 11]
```diff
@@ -165,15 +164,18 @@ def objective_closure():
...
+plt.fill_between(
+    xx[:, 0],
+    mean[:, 0] - 1.96 * np.sqrt(var[:, 0]),
+    mean[:, 0] + 1.96 * np.sqrt(var[:, 0]),
+    color="C0",
+    alpha=0.2,
+)
+
+plt.plot(xx, samples[:, :, 0].numpy().T, "C0", linewidth=0.5)
+plt.xlim(-0.1, 1.1)
```
Comment (awav) [inline doc/source/notebooks/basics/regression.pct.py]:
```suggestion
_ = plt.xlim(-0.1, 1.1)
```

File: doc/source/notebooks/tailor/mixture_density_network.pct.py [hunk 12]
```diff
@@ -28,28 +28,31 @@
...
 
+
 X, Y = data = sinusoidal_data(N, NOISE_STD)
 
-plt.plot(X, Y,'ro', alpha=0.3);
-plt.xlabel("$x$");
-plt.ylabel("$y$");
+plt.plot(X, Y, "ro", alpha=0.3)
+plt.xlabel("$x$")
+plt.ylabel("$y$")
```
Comment (awav) [inline doc/source/notebooks/tailor/mixture_density_network.pct.py]:
```suggestion
_ = plt.ylabel("$y$")
```

File: doc/source/notebooks/theory/Sanity_check.pct.py [hunk 13]
```diff
@@ -46,31 +46,51 @@
...
 # %%
 np.random.seed(0)
-X = np.random.rand(20,1)*10
-Y = np.sin(X) + 0.9 * np.cos(X*1.6) + np.random.randn(*X.shape)* 0.4
-Xtest = np.random.rand(10,1)*10
-plt.plot(X, Y, 'kx', mew=2);
+X = np.random.rand(20, 1) * 10
+Y = np.sin(X) + 0.9 * np.cos(X * 1.6) + np.random.randn(*X.shape) * 0.4
+Xtest = np.random.rand(10, 1) * 10
+plt.plot(X, Y, "kx", mew=2)
```
Comment (awav) [inline doc/source/notebooks/theory/Sanity_check.pct.py]:
```suggestion
_ = plt.plot(X, Y, "kx", mew=2)
```

File: doc/source/notebooks/theory/upper_bound.pct.py [hunk 14]
```diff
@@ -141,7 +147,7 @@ def plot_model(m, name=""):
 plt.axhline(full_lml, label="full", alpha=0.3)
 plt.xlabel("Number of inducing points")
 plt.ylabel("LML estimate")
-plt.legend();
+plt.legend()
```
Comment (awav) [inline doc/source/notebooks/theory/upper_bound.pct.py]:
```suggestion
_ = plt.legend()
```

File: doc/source/notebooks/understanding/models.pct.py [hunk 15]
```diff
@@ -260,13 +273,13 @@ def closure():
...
-p_test /= p_test.sum(1)[:,None]
+p_test /= p_test.sum(1)[:, None]
 
 # %%
 plt.figure(figsize=(12, 6))
 for i in range(3):
-    plt.contour(xx, yy, p_test[:,i].reshape(200,200), [0.5], colors='k', linewidths=1)
-plt.scatter(X[:,0], X[:,1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis);
+    plt.contour(xx, yy, p_test[:, i].reshape(200, 200), [0.5], colors="k", linewidths=1)
+plt.scatter(X[:, 0], X[:, 1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis)
```
Comment (awav) [inline doc/source/notebooks/understanding/models.pct.py]:
```suggestion
_ = plt.scatter(X[:, 0], X[:, 1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis)
```

File: doc/source/notebooks/tailor/mixture_density_network.pct.py [hunk 16]
```diff
@@ -218,9 +221,9 @@ def moon_data(N, noise):
 
 # %%
 X, Y = data = moon_data(N, NOISE_STD)
-plt.plot(X, Y, 'ro', alpha=0.3);
-plt.xlabel("$x$");
-plt.ylabel("$y$");
+plt.plot(X, Y, "ro", alpha=0.3)
+plt.xlabel("$x$")
+plt.ylabel("$y$")
```
Comment (awav) [inline doc/source/notebooks/tailor/mixture_density_network.pct.py]:
```suggestion
_ = plt.ylabel("$y$")
```

File: doc/source/notebooks/tailor/external-mean-function.pct.py [hunk 17]
```diff
@@ -64,7 +64,7 @@ def generate_data(num_functions=10, N=1000):
 Xs, F = generate_data(10)
 
 # %%
-plt.plot(Xs, F);
+plt.plot(Xs, F)
```
Comment (awav) [inline doc/source/notebooks/tailor/external-mean-function.pct.py]:
```suggestion
_ = plt.plot(Xs, F)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 18]
```diff
@@ -92,9 +94,9 @@ def invlink(f):
 Y_gen = rng.binomial(1, p)
 
 # plot
-plt.plot(x_grid, p_grid[:, 1], 'C1', linewidth=2)
-plt.plot(X_gen, p, 'C1o', ms=6)
-plt.plot(X_gen, Y_gen, 'C3x', ms=8, mew=2);
+plt.plot(x_grid, p_grid[:, 1], "C1", linewidth=2)
+plt.plot(X_gen, p, "C1o", ms=6)
+plt.plot(X_gen, Y_gen, "C3x", ms=8, mew=2)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(X_gen, Y_gen, "C3x", ms=8, mew=2)
```

File: doc/source/notebooks/basics/classification.pct.py [hunk 19]
```diff
@@ -39,11 +40,11 @@
...
 
 # %%
-X = np.genfromtxt('data/classif_1D_X.csv').reshape(-1, 1)
-Y = np.genfromtxt('data/classif_1D_Y.csv').reshape(-1, 1)
+X = np.genfromtxt("data/classif_1D_X.csv").reshape(-1, 1)
+Y = np.genfromtxt("data/classif_1D_Y.csv").reshape(-1, 1)
 
 plt.figure(figsize=(10, 6))
-plt.plot(X, Y, 'C3x', ms=8, mew=2);
+plt.plot(X, Y, "C3x", ms=8, mew=2)
```
Comment (awav) [inline doc/source/notebooks/basics/classification.pct.py]:
```suggestion
_ = plt.plot(X, Y, "C3x", ms=8, mew=2)
```

File: doc/source/notebooks/advanced/kernels.pct.py [hunk 20]
```diff
@@ -120,15 +124,15 @@ def plotkernelsample(k, ax, xmin=-3, xmax=3):
...
 X2 = np.linspace(-2, 2, 101).reshape(-1, 1)
 
 K21 = k(X2, X1)  # cov(f(X2), f(X1)): matrix with shape [101, 1]
-K22 = k(X2)      # equivalent to k(X2, X2) (but more efficient): matrix with shape [101, 101]
+K22 = k(X2)  # equivalent to k(X2, X2) (but more efficient): matrix with shape [101, 101]
 
 # plotting
 plt.figure()
-plt.plot(X2, K21);
+plt.plot(X2, K21)
```
Comment (awav) [inline doc/source/notebooks/advanced/kernels.pct.py]:
```suggestion
_ = plt.plot(X2, K21)
```

File: doc/source/notebooks/advanced/kernels.pct.py [hunk 21]
```diff
@@ -57,43 +59,44 @@
...
-axes[0,0].set_ylim(-3, 3);
+plotkernelsample(gpflow.kernels.Matern12(), axes[0, 0])
+plotkernelsample(gpflow.kernels.Matern32(), axes[0, 1])
+plotkernelsample(gpflow.kernels.Matern52(), axes[0, 2])
+plotkernelsample(gpflow.kernels.RBF(), axes[0, 3])
+plotkernelsample(gpflow.kernels.Constant(), axes[1, 0])
+plotkernelsample(gpflow.kernels.Linear(), axes[1, 1])
+plotkernelsample(gpflow.kernels.Cosine(), axes[1, 2])
+plotkernelsample(gpflow.kernels.Periodic(gpflow.kernels.SquaredExponential()), axes[1, 3])
+axes[0, 0].set_ylim(-3, 3)
```
Comment (awav) [inline doc/source/notebooks/advanced/kernels.pct.py]:
```suggestion
_ = axes[0, 0].set_ylim(-3, 3)
```

File: doc/source/notebooks/advanced/gps_for_big_data.pct.py [hunk 22]
```diff
@@ -214,8 +220,8 @@ def run_adam(model, iterations):
 
 logf = run_adam(m, maxiter)
 plt.plot(np.arange(maxiter)[::10], logf)
-plt.xlabel('iteration')
-plt.ylabel('ELBO');
+plt.xlabel("iteration")
+plt.ylabel("ELBO")
```
Comment (awav) [inline doc/source/notebooks/advanced/gps_for_big_data.pct.py]:
```suggestion
_ = plt.ylabel("ELBO")
```

File: doc/source/notebooks/advanced/gps_for_big_data.pct.py [hunk 23]
```diff
@@ -56,10 +58,10 @@ def func(x):
 # We plot the data along with the noiseless generating function:
 
 # %%
-plt.plot(X, Y, 'x', alpha=0.2)
+plt.plot(X, Y, "x", alpha=0.2)
 Xt = np.linspace(-1.1, 1.1, 1000)[:, None]
 Yt = func(Xt)
-plt.plot(Xt, Yt, c='k');
+plt.plot(Xt, Yt, c="k")
```
Comment (awav) [inline doc/source/notebooks/advanced/gps_for_big_data.pct.py]:
```suggestion
_ = plt.plot(Xt, Yt, c="k")
```

File: doc/source/notebooks/advanced/coregionalisation.pct.py [hunk 24]
```diff
@@ -128,40 +132,45 @@ def objective_closure():
...
-#  
+#
 # The covariance matrix between outputs is as follows:
 
 # %%
 B = coreg.output_covariance().numpy()
-print('B =', B)
-plt.imshow(B);
+print("B =", B)
+plt.imshow(B)
```
Comment (awav) [inline doc/source/notebooks/advanced/coregionalisation.pct.py]:
```suggestion
_ = plt.imshow(B)
```

File: doc/source/notebooks/advanced/coregionalisation.pct.py [hunk 25]
```diff
@@ -61,12 +62,12 @@
...
-Y1 = np.sin(6*X1) + np.random.randn(*X1.shape) * 0.03
-Y2 = np.sin(6*X2 + 0.7) + np.random.randn(*X2.shape) * 0.1
+Y1 = np.sin(6 * X1) + np.random.randn(*X1.shape) * 0.03
+Y2 = np.sin(6 * X2 + 0.7) + np.random.randn(*X2.shape) * 0.1
 
 plt.figure(figsize=(8, 4))
-plt.plot(X1, Y1, 'x', mew=2)
-plt.plot(X2, Y2, 'x', mew=2);
+plt.plot(X1, Y1, "x", mew=2)
+plt.plot(X2, Y2, "x", mew=2)
```
Comment (awav) [inline doc/source/notebooks/advanced/coregionalisation.pct.py]:
```suggestion
_  = plt.plot(X2, Y2, "x", mew=2)
```

File: doc/source/notebooks/theory/upper_bound.pct.py [hunk 26]
```diff
@@ -96,7 +101,7 @@ def plot_model(m, name=""):
...
```
Comment (awav) [inline doc/source/notebooks/theory/upper_bound.pct.py]:
```suggestion
_ = plt.legend()
```

File: doc/source/notebooks/understanding/models.pct.py [hunk 27]
```diff
@@ -202,55 +205,65 @@ def closure():
...
-plt.style.use('ggplot')
+
+plt.style.use("ggplot")
 # %matplotlib inline
 import matplotlib
-matplotlib.rcParams['figure.figsize'] = (12,6)
-plt.scatter(X[:,0], X[:,1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis);
+
+matplotlib.rcParams["figure.figsize"] = (12, 6)
+plt.scatter(X[:, 0], X[:, 1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis)
```
Comment (awav) [inline doc/source/notebooks/understanding/models.pct.py]:
```suggestion
_ = plt.scatter(X[:, 0], X[:, 1], 100, np.argmax(Y, 1), lw=2, cmap=plt.cm.viridis)
```

**Comment by awav** [comment]
> Black strips all the trailing `;` that we used in code cells. The reason we had those was that pyplot commands often return axes or other plotting objects, and notebook cells print the return value of the last executed line - so this litters up the notebooks. I don't think we should merge this without discussing what to do about all those cases.

Fixed the issue with `_ =`. It was a mixture of `;` and `_ =`. Btw, `;` it bad syntax, really easy to make an error with it.

Added the `setup.py` as suggested.

File: doc/source/notebooks/advanced/mcmc.pct.py [hunk 28]
```diff
@@ -380,77 +395,74 @@ def run_chain_fn():
...
+
+plt.figure(figsize=(8, 4))
 for hp in hyperparameters:
     plt.plot(samples[name_to_index[hp]], label=hp)
-plt.legend(bbox_to_anchor=(1., 1.))
-plt.xlabel('hmc iteration')
-plt.ylabel('hyper-parameter value');
+plt.legend(bbox_to_anchor=(1.0, 1.0))
+plt.xlabel("hmc iteration")
+plt.ylabel("hyper-parameter value")
```
Comment (st--) [inline doc/source/notebooks/advanced/mcmc.pct.py]:
```suggestion
_ = plt.ylabel("hyper-parameter value")
```

File: doc/source/notebooks/advanced/mcmc.pct.py [hunk 29]
```diff
@@ -474,13 +487,13 @@ def run_adam(model, iterations):
...
 maxiter = ci_niter(3000)
-logf = run_adam(model, maxiter) # start near Maximum a posteriori (MAP)
+logf = run_adam(model, maxiter)  # start near Maximum a posteriori (MAP)
 
 plt.plot(np.arange(maxiter)[::10], logf)
-plt.xlabel('iteration')
-plt.ylabel('neg_log_marginal_likelihood');
-
+plt.xlabel("iteration")
+plt.ylabel("neg_log_marginal_likelihood")
```
Comment (st--) [inline doc/source/notebooks/advanced/mcmc.pct.py]:
```suggestion
_ = plt.ylabel("neg_log_marginal_likelihood")
```

File: doc/source/notebooks/advanced/mcmc.pct.py [hunk 30]
```diff
@@ -535,44 +545,47 @@ def run_chain_fn():
...
+    np.percentile(rate_samples, 5, axis=0),
+    np.percentile(rate_samples, 95, axis=0),
+    color=line.get_color(),
+    alpha=0.2,
+)
 
-plt.plot(X, Y, 'kx', mew=2)
-plt.ylim(-0.1, np.max(np.percentile(rate_samples, 95, axis=0)));
+plt.plot(X, Y, "kx", mew=2)
+plt.ylim(-0.1, np.max(np.percentile(rate_samples, 95, axis=0)))
```
Comment (st--) [inline doc/source/notebooks/advanced/mcmc.pct.py]:
```suggestion
_ = plt.ylim(-0.1, np.max(np.percentile(rate_samples, 95, axis=0)))
```

File: doc/source/notebooks/advanced/mcmc.pct.py [hunk 31]
```diff
@@ -535,44 +545,47 @@ def run_chain_fn():
...
-plt.figure(figsize=(8,4))
+plt.figure(figsize=(8, 4))
 for hp in hyperparameters:
     plt.plot(parameter_samples[name_to_index[hp]], label=hp)
-plt.legend(bbox_to_anchor=(1., 1.))
-plt.xlabel('hmc iteration')
-plt.ylabel('hyper-parameter value');
+plt.legend(bbox_to_anchor=(1.0, 1.0))
+plt.xlabel("hmc iteration")
+plt.ylabel("hyper-parameter value")
```
Comment (st--) [inline doc/source/notebooks/advanced/mcmc.pct.py]:
```suggestion
_ = plt.ylabel("hyper-parameter value")
```

**Pull request merged**